app/controllers/iep_documents_controller.rb

```diff
@@ -0,0 +1,37 @@
+class IepDocumentsController < ApplicationController
+
+  before_action :find_iep_document, :authorize!
+
+  def show
+    @file_name = @iep_document.file_name
+    @bucket_name = ENV['AWS_S3_IEP_BUCKET']
+
+    object = s3.get_object({
```

Comment:
Yes it does. The AWS SDK automatically pulls in that config from ENV: http://docs.aws.amazon.com/sdkforruby/api/#Configuration